app/models/policy.rb

```diff
@@ -30,6 +30,16 @@ def sub_policy
   end
   alias_method :sub_policy?, :sub_policy
 
+  attr_accessor :organisation_content_ids
+  attr_accessor :people_content_ids
+  attr_accessor :working_group_content_ids
+
+  after_initialize do
+    self.organisation_content_ids = []
```

Comment:
Please change these to `||=` rather than `=`.

The current form prevents initializing the object with values set for these attributes. Not a use case I can see in the current state of the codebase but the behaviour is less surprising.